### ISSUE TITLE
Feature request: Always announce negative numbers in Excel

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#8875]
### Steps to reproduce:

In Excel, numbers (and any information) are reported by NVDA as they are displayed. Generally this is desirable.  One instance which may cause confusion is where negative numbers are indicated solely by colour.

1) Open Excel.
2) Type a negative value in a cell, eg -18
3) On that cell, press CONTROL+1 to open cell formatting.
4) On the "Number" page (the first page), press TAB to move to the number category.
5) Press TAB three times to move to the "Negative numbers" list.
6) Press DOWN ARROW once to the option which reads "1234.10" in read with no "-" preceding it, then press ENTER.
7) Read the cell with NVDA.

### Actual behavior:

NVDA reads "18", or, if "Colour" is checked in NVDA's document formatting, NVDA reads "Bright Red, 18".

### Expected behavior:

The user may expect that NVDA should identify that this is actually a negative number.  A sighted user still only has colour to identify that this is a negative value, which is less than ideal, but as many spreadsheets use various colour changes simply for appearance to indicate the region of the sheet rather than to convey meaning, the reason behind colour changes will not always be obvious to an NVDA user, even assuming they have set "colour" to be reported by NVDA automatically.

While it is possible another way may be used (such as the TEXT function) to display a number in this way, if NVDA could check for the final value of the cell and prepend "Negative" to it if this is not already indicated, that may avoid potential confusion for users.

#### NVDA Installed/portable/running from source:
Installed

#### NVDA version:
2018.3.2

#### Windows version:
10, v1809, build 17763.1

#### Name and version of other software in use when reproducing the issue:
Excel from Office 365 version 16.0.10827.20150 

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
No

**Comment by Brian1Gaff** [comment]
Yes, if you save something like this as static csv, one supposes it has to 
be converted back to - then at any rate.




bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.

**Comment by Qchristensen** [comment]
I can't answer the question about whether Microsoft expose the underlying number - to be honest, I think unless they do, this issue would require too much overhead.  And if they do, then we would need to ensure we still read other numbers as they were displayed, for instance, underneath the formatting, times and dates are actually just numbers - if you insert the date 24/10/2018, then change the cell to display it as a regular number, you get 43397.00, and we definitely wouldn't want that read in place of a date.

What I was going to add though, was if you set the number format to show in red with no negative symbol, then save the file as a CSV, it saves what is visible in the cell - that is every number is saved as a positive number.  The same happens if you select those cells, copy the data and paste it into say notepad.
